SmartDeviceLink/SDLSystemCapabilityManager.m

```diff
@@ -128,14 +136,15 @@ - (void)stop {
     _videoStreamingCapability = nil;
     _remoteControlCapability = nil;
     _seatLocationCapability = nil;
-    _appServicesCapabilitiesDictionary = [NSMutableDictionary dictionary];
 
     _supportsSubscriptions = NO;
-    [_capabilityObservers removeAllObjects];
-    [_subscriptionStatus removeAllObjects];
+    [self sdl_runSyncOnQueue:^{
+        self->_appServicesCapabilitiesDictionary = [NSMutableDictionary dictionary];
```

Comment:
Why are you using `->` instead of `.`?

Reply:
If you access the variable within its own getter using self., it causes infinite recursion.